app/src/org/commcare/android/view/c3/AxisConfiguration.java

```diff
@@ -0,0 +1,177 @@
+package org.commcare.android.view.c3;
+
+import org.commcare.android.util.InvalidStateException;
+import org.commcare.suite.model.graph.Graph;
+import org.commcare.suite.model.graph.GraphData;
+import org.commcare.suite.model.graph.SeriesData;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.Iterator;
+
+/**
+ * Axis-related configuration for C3.
+ *
+ * Created by jschweers on 11/16/2015.
+ */
+public class AxisConfiguration extends Configuration {
+    public AxisConfiguration(GraphData data) throws JSONException, InvalidStateException {
+        super(data);
+
+        JSONObject x = new JSONObject();
+        JSONObject y = new JSONObject();
+        JSONObject y2 = new JSONObject();

```

Comment:
An axis displayed on the right, with a different scale than the left/primary y axis

![temp](https://cloud.githubusercontent.com/assets/1486591/11223721/6f201124-8d3f-11e5-8375-3d7a93a30042.jpg)